src/core/attrsgenerator.cpp

```diff
@@ -41,19 +41,24 @@ AttrsGeneratorPtr AttrsGenerator::parse(const AttributesScope& attrsScope,
     int size = cmd.toInt(&sizeIsValid);
     if (sizeIsValid) { // cmd is just an interger
         _cmd = QString("*%1;min").arg(size);
+        cmds.push_back(cmd);
         cmds.push_back("min");
     } else {
-        cmds = cmd.split(";");
+        _cmd = cmd;
+        if (cmd.indexOf(";") != -1){            
+            cmds = _cmd.split(";");
+        } else {
+            cmds << _cmd;
+        }
```

Comment:
https://doc.qt.io/qt-5/qstring.html#split
> Splits the string into substrings wherever sep occurs, and returns the list of those strings. If sep does not match anywhere in the string, split() returns a single-element list containing this string.

So, we don't need to check for `;` here. If `split(';')` doesn't find any `;`, it will just return the `cmd` in a list with one element, which is exactly the same thing that you do here, but with one line of code.

Reply:
As mentioned above, my build resulted in a crash whenever it was forced to split by a non-existent character. I tried on a new build and the problem persists :/

Reply:
:thinking: That's very weird. Are you saying that the following code crashes?
``` c++
QString a = "abc";
auto b = a.split(";");
```
BTW, `maxs` works fine for me.

Reply:
I went back and tested on my other branches and it seems like it throws an error as expected, yet I'm positively sure I encountered this error when I started working on the project (before modifying the code, in fact, I initially thought something went wrong with the compilation). Again, I may be wrong on this, so if you believe that this change is unnecessary and only makes things more complicated, I'll remove it 😄

Reply:
@cardinot I think I re-encountered the bug I mentioned. Evoplex crashed when typing ``*1600``